pkg/utils/has/controller.go

```diff
@@ -507,21 +507,21 @@ func (h *SuiteController) WaitForComponentPipelineToBeFinished(component *appser
 				return false, nil
 			}
 
-			GinkgoWriter.Printf("PipelineRun %s reason: %s\n", pr.Name, pr.GetStatusCondition().GetCondition(apis.ConditionSucceeded).GetReason())
+			GinkgoWriter.Printf("PipelineRun %s/%s reason: %s\n", pr.GetNamespace(), pr.GetName(), pr.GetStatusCondition().GetCondition(apis.ConditionSucceeded).GetReason())
 
 			if !pr.IsDone() {
 				return false, nil
 			}
 
-			if pr.GetStatusCondition().GetCondition(apis.ConditionSucceeded).IsTrue() {
+			if utils.PipelineRunSucceeded(pr) {
```

Comment:
Can we name this function in such a way as to suggest that it returns a boolean value?
Something like: `isPipelineRunSucceeded(pr)`, `didPipelineRunSucceed(pr)`

Reply:
I chose the function name that looked natural to me in the context:

`if PipelineRunSucceeded()`
vs
`if isPipelineRunSucceeded()`
vs
`if didPipelineRunSucceed()`
(could be subjective though 😄  )

Reply:
I'd choose `isPipelineRunSucceeded` or `hasPipelineRunSucceeded`.
For plain `if` it could look a bit awkward, but from the name of the function we know that a `bool` value expected and it looks clear in complex conditions like:
```
if countPipelineRunsForComponent(component) == 1 && isPipelineRunSucceeded(pipelineRun) {
```

Reply:
`HasPipelineRunSucceeded` sounds ok to me, I haven't think of that one, thanks.

I'd still prefer `if PipelineRunSucceeded()`, but you're guys right that it should be clear that the function returns boolean value. In some cases `PipelineRunSucceeded` could be interpreted as a statement, while with the prefix `Has` it indicates that the output of the function should be true/false. I'll probably go with `HasPipelineRunSucceeded` then, as it also feels most natural in the context of `if`

To me prefix `Is` sounds good when it's used with adverb, i.e. `IsPipelineRunReady`. When it's used with a verb, like `if IsPipelineRunSucceeded`, just reading it makes me shiver, sorry, lol

Reply:
I agree, reading `IsPipelineRunSucceeded` more than 5 times in a row gave me a headache